src/format-files.js

```diff
@@ -64,7 +64,7 @@ async function formatStdin(prettierESLintOptions) {
   const stdinValue = (await getStdin()).trim()
   try {
     const formatted = format({text: stdinValue, ...prettierESLintOptions})
-    console.log(formatted)
+    console.error(formatted)
```

Comment:
You’re right. I did a mass replace under an assumption that `console.log` is a diagnostic tool. I would usually `process.stdout.write`, but there is nothing wrong with you using `console.log` for the main output, though.
Fixed.